src/System.IO.FileSystem.AccessControl/tests/DirectoryObjectSecurityTests.cs

```diff
@@ -91,25 +94,189 @@ public void RemoveAuditRuleAll_InvalidObjectAuditRule()
         }
 
         [Fact]
+        public void RemoveAuditRuleAll_Succeeds()
+        {
+            var descriptor = new CommonSecurityDescriptor(true, true, string.Empty);
+            var customObjectSecurity = new CustomDirectoryObjectSecurity(descriptor);
+
+            var objectTypeGuid = Guid.NewGuid();
+            var identityReference = new NTAccount(@"NT AUTHORITY\SYSTEM");
+
+            //https://msdn.microsoft.com/en-us/library/aa394063(v=vs.85).aspx
+            // read access mask  : 0x01
+            // write access mask : 0x02
+            var customAuditRuleReadWrite = new CustomAuditRule
+                (
+                    identityReference, 0x01 | 0x02, true, InheritanceFlags.None,
+                    PropagationFlags.None, objectTypeGuid, Guid.NewGuid(), AuditFlags.Success
+                );
+            var customAuditRuleSynchronize = new CustomAuditRule
+                (
+                    new NTAccount(@"NT AUTHORITY\SYSTEM"), 0x100000, true, InheritanceFlags.None,
```

Comment:
No, but If I am not mistaken, the value should represent a valid account in the system, otherwise an exception ( "IdentityNotMappedException" ) is thrown while modifying the access/audit rules.